operator/pkg/helmreconciler/reconciler.go

```diff
@@ -540,6 +555,59 @@ func CreateNamespace(cs kubernetes.Interface, namespace string, network string)
 	return nil
 }
 
+func (h *HelmReconciler) analyzeWebhooks(whs []string) error {
+	if len(whs) == 0 {
+		return nil
+	}
+
+	sa := local.NewSourceAnalyzer(istioConfigSchema.MustGet(), analysis.Combine("webhook", &webhook.Analyzer{
+		SkipServiceCheck: true,
+	}),
+		resource.Namespace(h.iop.Spec.GetNamespace()), resource.Namespace(istioV1Alpha1.Namespace(h.iop.Spec)), nil, true, 30*time.Second)
+	var localWebhookYAMLReaders []local.ReaderSource
+	var parsedK8sObjects object.K8sObjects
+	for _, wh := range whs {
+		k8sObjects, err := object.ParseK8sObjectsFromYAMLManifest(wh)
+		if err != nil {
+			return err
+		}
+		objYaml, err := k8sObjects.YAMLManifest()
+		if err != nil {
+			return err
+		}
+		whReaderSource := local.ReaderSource{
+			Name:   "",
+			Reader: strings.NewReader(objYaml),
+		}
+		localWebhookYAMLReaders = append(localWebhookYAMLReaders, whReaderSource)
+		parsedK8sObjects = append(parsedK8sObjects, k8sObjects...)
+	}
+	err := sa.AddReaderKubeSource(localWebhookYAMLReaders)
+	if err != nil {
+		return err
+	}
+
+	k := cfgKube.NewInterfaces(h.restConfig)
+	sa.AddRunningKubeSource(k)
+
+	// Analyze webhooks
+	res, err := sa.Analyze(make(chan struct{}))
+	if err != nil {
+		fmt.Println(err)
```

Comment:
Please remove this line as the error is already returned to the caller.

Reply:
This makes sense, I forgot to remove it
This was there for initial implementation